### DISCUSSION TITLE
Google gemin is not Working

### DISCUSSION THREAD

**Discussion by rishrust** [end-4/dots-hyprland#985] [Q&A]
i have already entered the api 
but the gemini doesnt replay i used the same key in my python script so the key is ok 
although openrouter api works please help

![image](https://github.com/user-attachments/assets/c5825538-4f09-4bb7-b455-4c769c5882fb)

**Comment by tuxmanxd** [comment]
change model to flash 1.5

**Reply by onlinemax** [reply]
How do you do that

**Reply by onlinemax** [reply]
Doesn't work
![image](https://github.com/user-attachments/assets/f7eced73-661e-4d13-99d6-2566955bd702)

**Reply by tuxmanxd** [reply]
did you type /key *yourapikeyhere* ?

**Reply by onlinemax** [reply]
Yeah of course tested the key even

**Comment by little-tonii** [comment]
same problem. First, I clear the chat console -> change to balance mode -> ask a question -> problem fixed. I dont know why but it worked for me
reboot maybe needed
remember, only use the flash model, I tried the pro model, it didnt work.
